src/networking.c

```diff
@@ -3047,6 +3059,7 @@ sds catClientInfoString(sds s, client *client) {
     if (client->flags & CLIENT_READONLY) *p++ = 'r';
     if (client->flags & CLIENT_NO_EVICT) *p++ = 'e';
     if (client->flags & CLIENT_NO_TOUCH) *p++ = 'T';
+    if (client->flags & CLIENT_REPL_RDB_CHANNEL) *p++ = 'C';
```

Comment:
no recommend, we should pay an attention for this change

Reply:
what do you mean by pay attenion? I mentioned that in the top comment etc. Oran requested this one. It also helps with testing, I can detect rdbchannel and kill it etc

Reply:
oh, please ignore my comments.